src/core.js

```diff
@@ -65,6 +65,11 @@ register([
     require('./locale-en-us')
 ]);
 
+// locales that are present in the window should be loaded
+if(window.PlotlyLocales && window.PlotlyLocales.length) {
```

Comment:
```js
if(window.PlotlyLocales && Array.isArray(window.PlotlyLocales)) {
// ...
}
```

would be safer. Note that `Plotly.register([])` is handled correctly (essentially a noop) at the moment.

Reply:
Done in f169380281377e3af40bb0772112d8ad9084f092